examples/neo-place/create-plugin.js

```diff
@@ -1,4 +1,4 @@
-const { Neo4jGraphQLSubscriptionsAMQPPlugin } = require("@neo4j/graphql-plugin-subscriptions-amqp");
+const { Neo4jGraphQLSubscriptionsEngineAMQP } = require("@neo4j/graphql-subscriptions-engine-amqp");
```

Comment:
Remember to deploy this to the live demo at https://team-graphql.uc.r.appspot.com/